app/src/main/java/com/example/tictacparty/MainActivity.kt

```diff
@@ -19,6 +19,7 @@ object GlobalVariables {
     var loggedInUser: String? = null
     var loggedIn : Boolean = false
     var player : Player? = null
+    var opponentPlayer : Player? = null
```

Comment:
Är det inte denna vi talat om inte skall finnas? (flera gånger) eftersom spelen hanteras i klassen/objektet "Game"?
Kanske är jag bara förvirrad.

Reply:
Ja vi var inte helt eniga , jag tycker det känns smidigt att använda för att slippa läsa ner från databasen två gånger på två olika platser(matchmaking and gameactivity) för informationen om den andra spelaren som man spelar mot.
 
Jag uppfattade det som att karl ej såg helt riktigt poängen, men det kändes inte som han var helt emot idén och samir sa att han tänkte att han skulle testa båda mina och karls idéer och jag trodde du sa "Kör på" och att du höll med i samtal om detta, men jag kanske hörde fel/missuppfatta! :) Sen sa jag även i vår diskussion, att det kanske får bli upp till den som programmerar och att välja vad som är smidigast för den kod som personen skriver, så länge vi får det resultatet vi alla vill ha. Jag tycker personligen att den underlättar! 

Vad känner du själv?